docs/testing.md

```diff
@@ -40,6 +40,10 @@ To run just a subset of them, specify a particular file or directory:
     py.test tests/unit
     py.test tests/unit/api.py
 
+To run just a pep8 test:
+
+    py.test --pep8 -m pep8
```

Comment:
If what you're trying to do is avoid `.venv`, you could just do `py.test --pep8 -m pep8 *`, which would pick up new directories, as well as e.g. `setup.py`